docs/swagger.yaml

```diff
@@ -0,0 +1,59 @@
+basePath: /
+definitions:
+  controllers.StatusResponse:
+    properties:
+      message:
+        type: string
+    type: object
+  entities.User:
+    properties:
+      email:
+        type: string
+      password:
+        type: string
+    type: object
+  server.Health:
+    properties:
+      status:
+        type: string
+    type: object
+host: localhost:8080
+info:
+  contact: {}
+  description: This is the babeltower HTTP API documentation.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  title: Babeltower API
+  version: "1.0"
+paths:
+  /healthcheck:
+    get:
+      produces:
+      - application/json
+      responses:
+        "200":
+          description: OK
+          schema:
+            $ref: '#/definitions/server.Health'
+      summary: Verify the service health
+  /users:
+    post:
+      consumes:
+      - application/json
+      parameters:
+      - description: User e-mail and password
+        in: body
+        name: user
+        required: true
+        schema:
+          $ref: '#/definitions/entities.User'
+      produces:
+      - application/json
+      responses:
+        "201":
```

Comment:
Add fail responses too

Reply:
For the `/healthcheck` endpoint, I don't know if I need to map a specific error.